src/applications/disability-benefits/2346/schemas/address-schema.js

```diff
@@ -148,7 +147,19 @@ export const addressUISchema = (
 
   return (function returnAddressUI() {
     return {
-      'view:livesOnMilitaryBase': {
+      'ui:order': [
```

Comment:
This function is for returning an address - why are we putting inside a ui object for an order?

Reply:
Because the ui elements are created here and we need them to return in the correct order.

Reply:
this is in address schema, though - why are we doing this here instead of say 2346UI?

Reply:
It does work if you put it in `2346UI` but you would have to repeat the `ui:order` for each property that's using `address-schema`. Having it inside of `address-schema` keeps it DRY.

Reply:
It doesn't make sense to have this be a part of the address - There are other properties in 2346UI that are their own schema - they're only in there once - and they remain available on all of the chapters.

Reply:
The other properties are not using a helper function to build a majority of their ui schema so their ui schemas should be placed in `2346UI`. A high percentage of the address properties ui schemas are being built by the `address-schema` helper and my opinion is that anything specific to each address property should be placed directly in 2346UI. Having said that, I do see an opportunity for more refactoring with placing the `ui:subtitle` and `ui:field` values in the helper since the same values are being used on the permanent address and temporary address properties. If you want to talk about getting rid of the helper and putting that ui schema inside `2346UI`, I'd be open to that.

Reply:
I get what you're saying - Its definitely easier to put it with the address, but that doesn't mean it belongs with the address.  The order is its own schema - its not a part of the address.

Reply:
I'm not sure I understand what you mean about the order being its own schema. It's directly manipulating the address properties.

Reply:
The order does belong in the main ui schema - not in the address schema, or the address helper or anything else address related.  You've made a decision to tightly couple the order with the address by including it in this helper - where it does not belong.

Reply:
I see - this was confusing because you've renamed `selectedProducts` to `order` and then added `ui:order` thus my confusion - they're separate - resolving this.